support-frontend/.husky/pre-commit

```diff
@@ -2,3 +2,11 @@
 . "$(dirname "$0")/_/husky.sh"
 
 cd support-frontend && yarn lint-staged
+
+if git diff-index --cached  --diff-filter=AM --name-only  HEAD | grep --quiet .scala$
+then
+  echo "Running scalafmtCheckAll"
+  cd ..
+  sbt scalafmtCheckAll
```

Comment:
Would it be possible/worth it to change from checking the formatting to actually performing the formatting? I can see it might be annoying, but my next step after this fails will always be manually running the formatting, so getting the hook to do it would save me time (though I’ll probably still need to stage the changes – I guess that provides a bit of safety in case I didn’t want the formatting).

Reply:
Ok I've updated it to run the format if the check fails